SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -350,5 +321,8 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	bool bUseSpatialView;
 
 public:
-	mutable FOnWorkerTypesChanged OnWorkerTypesChangedDelegate;
+	// UI Hidden settings passed through from SpatialGDKEditorSettings
+	bool bUseDevelopmentAuthenticationFlow;
+	FString DevelopmentAuthenticationToken;
+	FString DevelopmentDeploymentToConnect;
```

Comment:
git being weird?

Reply:
yup, that's me doing a bad merge with the mobile auth flow changes. i'll remove.